components/mpas-ocean/cime_config/buildnml

```diff
@@ -616,7 +618,7 @@ def buildnml(case, caseroot, compname):
             lines.append('        filename_interval="00-01-00_00:00:00"')
             lines.append('        clobber_mode="truncate"')
             lines.append('        reference_time="01-01-01_00:00:00"')
-            lines.append('        output_interval="none">')
+            lines.append('        output_interval="00-00-01_00:00:00">')
```

Comment:
I don't think we want to make this change.

Reply:
Thanks for flagging - there are a number of streams settings I've committed for convenience but I will need to remove once the draft version of this PR is complete.  I've separated such changes in commits with obvious names.